test/client/components/__snapshots__/Place.test.js.snap

```diff
@@ -6,38 +6,16 @@ exports[`With Jest snapshot, Place component renders rating section when present
 >
   <div
     className="name"
-  />
+  >
```

Comment:
did you forget to change your place test based on your code changes? as i said, it's your responsibility to make sure the updated snapshot is correct

Reply:
Okay I'm pretty new to snapshot but will do this properly